contributing/contributing-maui.md

```diff
@@ -2,15 +2,15 @@
 
 Thank you for your interest in contributing to the .NET MAUI Community Toolkit documentation.
 
-Documentation is **required** when completing a proposal as part of our [proposal workflow](https://github.com/CommunityToolkit/Maui/projects). 
+Documentation is **required** when completing a proposal as part of our [proposal workflow](https://github.com/CommunityToolkit/Maui/projects).
 
 ## Starting your changes
 
 You will need to fork this repository. We strongly recommend that you create a branch from your `main` branch so that it will reduce any complications when submitting your Pull Request.
 
 ## Recommended tooling
 
-We recommend that you use Visual Studio Code for the editing of documentation files along with the [Docs Authoring Pack](https://marketplace.visualstudio.com/items?itemName=docsmsft.docs-authoring-pack) extension. This helps with validating code example formatting and also spell checking among other rules, for more information check out the [Microsoft documentation](https://docs.microsoft.com/contribute/how-to-write-docs-auth-pack)
+We recommend that you use Visual Studio Code for the editing of documentation files along with the [Docs Authoring Pack](https://marketplace.visualstudio.com/items?itemName=docsmsft.docs-authoring-pack) extension. This helps with validating code example formatting and also spell checking among other rules. For more information, see the [Docs Authoring Pack documentation](/contribute/how-to-write-docs-auth-pack)
```

Comment:
This may not be relevant to docutune, but periods are inconsistent in this article. Missing at the end of this paragraph and throughout the 'Making your changes' section.